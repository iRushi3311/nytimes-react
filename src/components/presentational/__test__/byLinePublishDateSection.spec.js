import React from 'react';
import renderer from 'react-test-renderer';
import ByLinePublishTimeSection from '../byline-publish-date-section.js';

describe('>>>C O M P O N E N T S --- snapshot test for ByLinePublishTimeSection', () => {

    let props,
        componentTree;

    beforeEach(() => {
        props = {
            by: 'RPatel',
            publishdate: new Date('2017-08-17T09:00:01+0000')
        };
        componentTree = renderer.create(<ByLinePublishTimeSection {...props}/>).toJSON();
    });

    it('+++ ByLinePublishTimeSection match with snapshot ', () => {
        expect(componentTree).toMatchSnapshot();
    });

});