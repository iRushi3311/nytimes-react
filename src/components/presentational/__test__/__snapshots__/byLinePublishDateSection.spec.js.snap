// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>C O M P O N E N T S --- snapshot test for ByLinePublishTimeSection +++ ByLinePublishTimeSection match with snapshot  1`] = `
<h5
  style={
    Object {
      "marginBottom": "0px",
    }
  }
>
  <span
    className="author-info"
  >
    RPatel
  </span>
    
  <span
    className="published-time"
  >
    August 17  - 05:00 AM
  </span>
</h5>
`;
